scripts/redux/actions.js

```diff
@@ -20,3 +26,14 @@ const routeActions = {
     });
   }
 };
+
+const ticketsActions = {
```

Comment:
With this solution we have cleaner code, what about offline I don't know right now, we will play around it later